torch/csrc/utils/python_dispatch.cpp

```diff
@@ -345,6 +345,64 @@ void initDispatchBindings(PyObject* module) {
     return at::isTensorSubclassLike(tensor);
   });
 
+  m.def("_dispatch_total_keys", []() {
+    return (int64_t)c10::DispatchKey::EndOfFunctionalityKeys;
+  });
```

Comment:
This is a little bit of a distortion, because c10::DispatchKey has more entries than just the functionality keys as it also cross-products functionality key and backend components. Well you don't seem to be using it in this PR anyhow.

Reply:
Removed!